ShippingApp/src/main/java/com/javaschool/queue/ConfigRabbit.java

```diff
@@ -0,0 +1,37 @@
+package com.javaschool.queue;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ConfigRabbit {
+
+    @Value("${config-rabbit.queue}")
```

Comment:
If you notice here, all three values are under `config-rabbit` you can use the `@ConfigurationProperties` annotation instead.

Reply:
The values were changed to @ConfigurationProperties.